CMakeLists.txt

```diff
@@ -4,12 +4,38 @@ if(DEFINED ENV{VCPKG_ROOT} AND NOT DEFINED CMAKE_TOOLCHAIN_FILE)
     set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
         CACHE STRING "")
 endif()
-
 # Name of the project
 project (pc-ble-driver-py)
-
+#set(CMAKE_FIND_DEBUG_MODE 1)
+set(PYTHON_LIBRARY_FROM_USER_FLAGS "${PYTHON_LIBRARY}")
+# Use nont-debug version of python, even when building a debug version 
+add_definitions(-DSWIG_PYTHON_INTERPRETER_NO_DEBUG)
+message("CMAKE executable path ${CMAKE_COMMAND}")
+message("CMake Generator name: ${CMAKE_GENERATOR}")
```

Comment:
https://cmake.org/cmake/help/latest/command/message.html
It would be nice to specify type of message. Verbose would be appropriate I think.